apple-pie/index.html

```diff
@@ -4,36 +4,85 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">
+
+  <!-- Remember to link your styles -->
+  
   </head>
   <body>
-    Apple Pie
-
-    This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! 
     
-    Ingredients 
-    
-    1 recipe pastry for a 9 inch double crust pie 
-    1/2 cup unsalted butter 
-    3 tablespoons all-purpose flour
-    1/4 cup water
-    3 tablespoons all-purpose flour
-    1/4 cup water 1/2 cup white sugar
-    1/2 cup packed brown sugar
-    8 Granny Smith apples - peeled, cored and sliced
+    <div class = "containerApplePiePhoto">
+      <div class="centeredApplePie">Apple Pie</div>
+      <img class="ApplePiePhoto" src="images/apple-pie.jpg"> 
```

Comment:
```suggestion
      <img class="apple-pìe-photo" src="images/apple-pie.jpg"> 
```
No esta mal eeee!!, pero para css se usa mejor los guiones mas que CamelCase, es un tema de que se ha extendido dentro del mundillo, pero se puede usar ambos, pero queda mejor y es mas cercano a la programacion real si te vas aconstumbrando a ponerlo asi vale?

Reply:
¡Sí! Tengo que ir adaptándome a la forma de escritura porque tengo demasiadas manías xDD JAJAJA.  
¡Gracias!